libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/IDPAccountPickerActivity.java

```diff
@@ -53,13 +53,15 @@
 public class IDPAccountPickerActivity extends AccountSwitcherActivity {
 
     private static final int IDP_LOGIN_REQUEST_CODE = 999;
+    private static final String FEATURE_APP_IS_IDP = "IP";
     private static final String TAG = "IDPAccountPickerActivity";
 
     private SPConfig spConfig;
 
     @Override
     public void onCreate(Bundle savedInstance) {
         super.onCreate(savedInstance);
+        SalesforceSDKManager.getInstance().registerUsedAppFeature(FEATURE_APP_IS_IDP);
```

Comment:
Is it (dis)allowed to be SP and IDP at the same time?

Reply:
Not explicitly, no. Technically it would be extra hops but should still work fine ;-)